### PR TITLE
Fix more id1 issues

### PR DESCRIPTION
**Pull request by Hacki95**
Also removed modelid from some tables. There are probably a few more remaining.

closes https://github.com/azerothcore/mod-progression-system/issues/148

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
added "closes https://github.com/azerothcore/mod-progression-system/issues/148 "

File: src/Bracket_61_64/sql/world/progression_61_64_creature.sql [hunk 1]
```diff
@@ -1,76 +1,76 @@
...
-(4052,12379,0,0,0,1,1,13952,1,-11174.1,-2178.27,22.7358,3.31364,600,5,0,6761,0,1,0,0,0,'',0),
-(4053,7370,0,0,0,1,1,4870,0,-11172.6,-2170.94,22.7176,6.21176,600,5,0,5233,2991,1,0,0,0,'',0),
-(4054,7370,0,0,0,1,1,4870,0,-11173.4,-2199.74,22.7357,6.23542,600,5,0,5233,2991,1,0,0,0,'',0),
-(4055,12377,0,0,0,1,1,11649,0,-11184.5,-2080.05,35.505,1.53675,600,3,0,2699,0,1,0,0,0,'',0),
-(4056,12378,0,0,0,1,1,146,0,-11214.5,-2173.24,22.7096,2.86285,600,5,0,4580,6882,1,0,0,0,'',0),
-(4057,7370,0,0,0,1,1,4870,0,-11212.4,-2193.44,22.7397,2.30048,600,5,0,5233,2991,1,0,0,0,'',0),
-(4058,12377,0,0,0,1,1,11649,0,-11224.6,-2197.94,22.7143,3.35942,600,5,0,2699,0,1,0,0,0,'',0),
-(4059,12377,0,0,0,1,1,11649,0,-11240.2,-2197.94,22.7016,3.9263,600,5,0,2699,0,1,0,0,0,'',0),
-(4060,12377,0,0,0,1,1,11649,0,-11190.6,-2152.13,22.6994,2.11148,600,3,0,2699,0,1,0,0,0,'',0);
+INSERT INTO creature (`guid`,`id1`,`map`,`zoneId`,`areaId`,`spawnMask`,`phaseMask`,`equipment_id`,`position_x`,`position_y`,`position_z`,`orientation`,`spawntimesecs`,`wander_distance`,`currentwaypoint`,`curhealth`,`curmana`,`MovementType`,`npcflag`,`unit_flags`,`dynamicflags`,`ScriptName`,`VerifiedBuild`) VALUES
```
Comment (Helias) [inline src/Bracket_61_64/sql/world/progression_61_64_creature.sql:2]: I don't see a DELETE FROM before this, is there any reason?

File: src/Bracket_61_64/sql/world/progression_61_64_creature.sql [hunk 2]
```diff
@@ -1,76 +1,76 @@
 -- restore karazhan npc spawns
```
Comment (Helias) [inline src/Bracket_61_64/sql/world/progression_61_64_creature.sql:1]:
```suggestion
-- restore karazhan npc spawns
DELETE FROM `creature` WHERE `guid` IN (3961,3965,3966,3967,3968,3976,3977,3978,3979,3986,3987,3988,3989,4000,4001,4002,4003,4004,4005,4006,4007,4008,4009,4011,4012,4013,4014,4015,4016,4017,4018,4019,4020,4021,4022,4023,4024,4025,4026,4027,4028,4029,4030,4031,4032,4033,4034,4035,4036,4040,4041,4042,4043,4044,4045,4046,4047,4049,4050,4051,4052,4053,4054,4055,4056,4057,4058,4059,4060);
```

**Comment by xclid** [comment]
```
Loading world information...
> RealmID:              1
> Version DB world:     ACDB 335.6-dev
>> Applying update "progression_80_9_creature.sql" '2F621D1'...
>> Applying update "progression_80_9_disables.sql" '87F2403'...
Will clear `logs` table of entries older than 1209600 seconds every 10 minutes.
Using DataDir ../data/
```

Tested and works fine for me.

**Comment by Helias** [comment]
@xclid thanks for testing, but if you import the sql twice you should get an issue like "duplicate guid/entry" because of the missing `DELETE` query

**Pull request closed without merging**